doc/classes/EditorExportPlugin.xml

```diff
@@ -108,6 +108,7 @@
 			<return type="void" />
 			<argument index="0" name="path" type="String" />
 			<argument index="1" name="tags" type="PackedStringArray" />
+			<argument index="2" name="arg2" type="String" />
```

Comment:
The name should be added to the `bind_method` call.

Reply:
Done.